code/game/objects/items/twohanded.dm

```diff
@@ -869,7 +873,8 @@
 	user.visible_message("[user] lowers [src].","You lower [src].")
 	item_state = "binoculars"
 	user.regenerate_icons()
-	if(user && user.client)
+	var/mob/living/carbon/human/H = user
```

Comment:
ought to check that the mob actually is in fact a human first

Reply:
what, for monkeys? don't think anything else would be picking it up

Reply:
dexterous stands are one example, but there is a proc for simplemobs to hold items

Reply:
it's as easy as doing `iscarbon(user)` before doing this, and putting the DNA check in a separate if() statement

Reply:
and yeah, monkeys, stands, drones etc would all break this

Reply:
how do you like this